sources/Application/Instruments/SamplePool.cpp

```diff
@@ -166,7 +166,9 @@ bool SamplePool::loadSample(const char *path) {
   WavFile *wave = WavFile::Open(path);
   if (wave) {
     wav_[count_] = wave;
-    const std::string name = wavPath.GetName();
+    std::string name = wavPath.GetName();
+    // Drop file extension
+    name.resize(name.length() - 4);
```

Comment:
This is much better :)
I think we should make it a bit more generic:
`
size_t index = name.find_last_of("."); 
name = name.substr(0, index); 
`

Reply:
Yeah, that does sound like a better idea... I've implemented that now.. with a little catch to make sure what is returned is valid